.env.example

```diff
@@ -23,6 +23,11 @@ REDIS_URI=
 # Optional: the JSON Google Application Credentials used for gcloud.driver
 # GOOGLE_APPLICATION_CREDENTIALS=
 
+# Optional: the Hetzner Cloud API token used for hetznercloud.driver
+# HETZNER_API_KEY=
+# the Hetzner Cloud image ID used for portal creation. Setup guide at https://gist.github.com/69/a60838a5d67807da574edcfd3193af64
```

Comment:
This should be separated into two lines, adding the capitalization to the first one:

```
# The Hetzner Cloud image ID used for portal creation.
# Setup guide at https://gist.github.com/69/a60838a5d67807da574edcfd3193af64
```

Also that the guide needs updating to run into a limited user (adduser, add it to docker group etc.). Probably should be moved into Cryb docs in the future when we do them?

Reply:
Also, this is probably more personal than anything, but I feel it could be done either way via /etc/rc.local, and making sure the script runs there. Just in case of anything, even if it'll be contained...

Reply:
I looked into it, and it seems /etc/rc.local is officially deprecated in Debian 10 and I feel like it'd be more effort adding it back than just adding a line to crontab.

Reply:
Alright, that's minimal - but still can we do the limited user thing?

Reply:
Yep, I'm working on it as we speak.